app/controllers/IndexController.php

```diff
@@ -1,9 +1,19 @@
 <?php
 
+namespace GanttDashboard\App\Controllers;
+
 class IndexController extends ControllerBase
 {
     public function indexAction()
     {
-        $this->session->set('worker_session', 1);
+    }
+
+    /**
+     * Define the notFound404 action
+     */
+    public function notFound404Action()
```

Comment:
I am pretty sure that the framework has a way to create 404 pages, and is definitely not like this

Reply:
I made that following phalcon's docs : https://docs.phalconphp.com/uk/3.3/dispatcher#handling-404
The xxx404 action must be made "by hand". Also the 
    $router->notFound([
        "controller" => "index",
        "action" => "route404"
    ]);
Doesn't seem to work.